geometry_msgs/Pose Pose
geometry_msgs/Vector3 rpy
bool use_rpy
---
bool success
string message